### ISSUE TITLE
Accessibility: No visible focus

### ISSUE THREAD

**Issue by nedredmond** [inklewriter/freeinklewriter#75]
For the few elements that are focusable by keyboard at the time of writing (anchor links), it appears that the browser focus outline has been deliberately removed and replaced with nothing. That means that users navigating by keyboard have no way of knowing where they are on the page.

Suggested fix:
Remove the CSS that sets `outline: 0` (and `:focus {outline: 0}`) to allow the browser-native accessible focus styles to function. OR, implement an alternative focus outline.

ADA Standards Violations:
- https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html

<img width="1513" height="967" alt="Image" src="https://github.com/user-attachments/assets/056d2746-16c7-423b-b186-6ae75a7f5b9f" />

**Comment by nedredmond** [comment]
I see [where in the stylesheets focus is overridden](https://github.com/inklewriter/freeinklewriter/blob/fe54e8d0603dcd507d3638437d2ab29ceb15f1c3/assets/stylesheets/reset.css.scss#L45-L48), adorably commented

```scss
/* remember to define focus styles! */
:focus {
	outline: 0;
}
```
